crates/rpc/rpc/src/eth/api/call.rs

```diff
@@ -40,6 +40,93 @@ where
         BlockReaderIdExt + ChainSpecProvider + StateProviderFactory + EvmEnvProvider + 'static,
     Network: NetworkInfo + Send + Sync + 'static,
 {
+    /// Executes   complex RPC calls to Ethereum nodes
+
+    pub async fn eth_multicall_v1(
+        &self,
+        multicall_bundle: MulticallBundle,
+        state_context: Option<StateContext>,
+        mut state_override: Option<StateOverride>,
+    ) -> EthResult<Vec<EthCallResponse>> {
```

Comment:
isn't this exactly like `call_many`?

Reply:
> isn't this exactly like `call_many`?

yes correct, should i change something else?

Reply:
I think we can just reuse the existing call_many impl then we also don't need the new MlticallBundle type

Reply:
> I think we can just reuse the existing call_many impl then we also don't need the new MlticallBundle type

no problem, will do that, thank you so much for your review!

Reply:
> I'm still not quite sure how this call differs from `callMany` then, but we can start with being just the same maybe we need more overrides I guess but for now this should be ok.

yes i agree with you, since it's a prototype for now, we can do it like this and upgrade it :) ty lot!